### PR TITLE
MAINT: Add manifest including versioneer

### LINKED ISSUE #24: CI: pypi deployment failing
@effigies have you ever seen something like:

```
Processing ./dist/sdcflows-0.1.0.tar.gz
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-v9krq8c7/setup.py", line 26, in <module>
        main()
      File "/tmp/pip-req-build-v9krq8c7/setup.py", line 13, in main
        import versioneer
    ImportError: No module named 'versioneer'
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-req-build-v9krq8c7/
```

### PR DESCRIPTION
**Pull request by effigies**
Closes #24.

### CODE REVIEW

**Comment by effigies** [comment]
@oesteban Will add a test deploy job.

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,8 @@
+include CHANGES.rst
+include LICENSE
+
+recursive-include sdcflows/data *
+
+# versioneer
+include versioneer.py
+include fmriprep/_version.py
```
Comment (oesteban) [inline MANIFEST.in]:
```suggestion
include sdcflows/_version.py
```

**Comment by oesteban** [comment]
You mean you'll split the test from the build? Because the deploy job includes some testing.

**Comment by effigies** [comment]
> You mean you'll split the test from the build? Because the deploy job includes some testing.

I'll do what we do in the other packages, and duplicate the deploy job minus the deploy step.

**Comment by effigies** [comment]
Packaging is working, assuming you've got PyPI credentials all set up. https://circleci.com/gh/poldracklab/sdcflows/23

**Review by oesteban — APPROVED** [review]

**Pull request merged**